src/components/OcButton.spec.js

```diff
@@ -0,0 +1,53 @@
+import OcButton from "./OcButton.vue";
+import {createLocalVue, shallowMount} from "@vue/test-utils";
+const localVue = createLocalVue()
+
+
+describe("oc button", () => {
+  const testSlots = { default: '<p class="text">Test button</p>' }
+  function getWrapperWithProps(props) {
+    return shallowMount(OcButton, { localVue: localVue, propsData: props })
+  }
+  function getWrapperWithTestSlot() {
+    return shallowMount(OcButton, { localVue: localVue, slots: testSlots })
+  }
+
+  describe("emit click event", () => {
+    it("emits click event when click is triggered", () => {
```

Comment:
Please use `async` and `await` as much as possible. This means:
- `async () => {` instead of `() => {`
- `await wrapper.vm.$nextTick()`
- the `expect` statement belongs on the top level of the function then

Reply:
Same for the other tests